dali/python/CMakeLists.txt

```diff
@@ -26,6 +26,15 @@ target_link_libraries(${dali_python_lib} PRIVATE ${CUDA_LIBRARIES} ${dali_lib})
 set_target_properties(${dali_python_lib} PROPERTIES
   LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/dali/python/nvidia/dali)
 
+string(TIMESTAMP TODAY "%Y%m%d")
```

Comment:
This basically makes the build non-repeatable after one day, still better that adding a full timestamp.

Still, if we want to use this, why not include it in all builds for clarity?

Otherwise -DINCLUDE_TIMESTAMP_IN_VERSION?

Reply:
Because we don't want to have 0.9-date, while we do for 0.9deve-date.
I agree that INCLUDE_TIMESTAMP_IN_VERSION may do.